src/comment.ts

```diff
@@ -59,9 +59,11 @@ export function renderComment({
   // Build footer
   let footer = ''
   if (includeFooter === undefined || includeFooter === true) {
-    footer =
-      `\n\n---\n\n_Triggered by @${github.context.actor},` +
-      ` Commit: \`${(github.context.payload as PullRequestEvent).pull_request.head.sha}\`_`
+    let commitInfo = ''
+    if ('pull_request' in github.context.payload) {
```

Comment:
Can we use the same check as in the other file here?
```suggestion
    if (github.context.eventName === 'pull_request') {
```